status-panels-lib/README.md

```diff
@@ -0,0 +1,103 @@
+# Status panels lib
+
+This status panels lib can be used to add a status panel row for integration dashboards using [grafonnet](https://github.com/grafana/grafonnet).
```

Comment:
Is it clear what status we are referring to (i.e How does this help the viewer/operator of a SUO)?

Reply:
I can add a bit more explanation but I think the examples below and screenshot should help.

Reply:
Extended the explanation.